### ISSUE TITLE
Request for an "Edit Code" Button in Index File for PIPELINES_VSCODE_REMOTES Variable

### ISSUE THREAD

**Issue by xsa-dev** [lucafaggianelli/plombery#252]
Description:
For enhanced editing convenience, it would be highly beneficial to include a button for editing code within the index file related to the PIPELINES_VSCODE_REMOTES variable. This button should enable users to easily navigate to the corresponding code in the repository, accessible both from the index file and the pipeline.

Example URL ([link](https://github.com/microsoft/vscode-remote-release/issues/4779#issuecomment-1220475865)):
```
vscode://vscode-remote/ssh-remote+alxy@alxy-ms-7640.alekseysavin.net:22/home/alxy/Data/plombery/examples/src/sales_pipeline_copy.py:1:1
```

This feature would significantly improve the workflow and streamline the process of code editing within the context of the specified variable. Your consideration of this enhancement request is greatly appreciated.

**Comment by lucafaggianelli** [comment]
Yes love the idea, I have a similar issue already open for linking files in a trace back error.

By the way where would you like to add the button? In the log view, so you can open the file where a log happened? Or in the tasks lists of a pipeline? Or a bit everywhere?

**Comment by xsa-dev** [comment]
I think this might not be crucial, but for the out-of-the-box version, it could be a button-link to the pipeline folder. In more detailed views, we could provide deeper links, such as a link to a specific pipeline or even a specific row and column (if there are rows other than 1:1).
Just an example, somewhere near the "Run" button, and links in the logs to specific lines ([I think we can add line outputs to the logger](https://stackoverflow.com/a/44401529)).
![image](https://github.com/lucafaggianelli/plombery/assets/16959353/b6b77d94-e33e-409d-88bd-a54437c0ae02)
